samples/client/petstore/python-experimental/openapi_client/api_client.py

```diff
@@ -23,9 +23,13 @@
 import six
```

Comment:
@wing328 do you know why this folder and file are generated?
We pass in `--additional-properties packageName=petstore_api' when generating the samples

Reply:
Please see https://github.com/OpenAPITools/openapi-generator/pull/2757/files#r326855503

Reply:
This issue has been resolved by my latest update. Only the petstore_api is now generated